plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -490,16 +490,23 @@ public class FeedReader.ttrssAPI : GLib.Object {
 					if(attachments != null)
 						mediaCount = attachments.get_length();
 
-					if(mediaCount > 0)
-					 	media = new Gee.ArrayList<string>();
 					for(int j = 0; j < mediaCount; ++j)
 					{
 						var attachment = attachments.get_object_element(j);
-						if(attachment.get_string_member("content_type").contains("audio")
-						|| attachment.get_string_member("content_type").contains("video"))
+						EnclosureType type = EnclosureType.FILE;
+						if(attachment.get_string_member("content_type").contains("audio"))
 						{
-							media.add(attachment.get_string_member("content_url"));
+							type = EnclosureType.AUDIO;
 						}
+						else if(attachment.get_string_member("content_type").contains("video"))
+						{
+							type = EnclosureType.VIDEO;
+						}
```

Comment:
I wonder if we should add an `EnclosureType.from_string` function to do this?
```
static EnclosureType from_string(string str)
{
  if (str.contains("audio"))
    return AUDIO;
  else if (str.contains("video"))
    return VIDEO;
  else
    return FILE;
}
```
?